test/EIP20Gateway/getLatestAnchorInfo.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const chai = require('chai');
+const Web3 = require('web3');
+const sinon = require('sinon');
+const EIP20Gateway = require('../../src/ContractInteract/EIP20Gateway');
+const Anchor = require('../../src/ContractInteract/Anchor');
+const SpyAssert = require('../../test_utils/SpyAssert');
+
+const assert = chai.assert;
+
+describe('EIP20Gateway.getLatestAnchorInfo()', () => {
```

Comment:
Like above comment this function should go to anchor class.

Reply:
I think we should have both. `Anchor.getLatestInfo()` and `EIP20Gateway.getLatestAnchorInfo()` that retrieves the info for the anchor it knows about.

Reply:
Added for both.